.github/workflows/api-dotnetcore.yml

```diff
@@ -71,17 +71,20 @@ jobs:
           cd ..
           head TestResults/coverage.opencover.xml
       - name: Codecov
-        uses: codecov/codecov-action@v1.0.7
-        with:
-          # User defined upload name. Visible in Codecov UI
-          name: PIMS
-          # Repository upload token - get it from codecov.io. Required only for private repositories
-          token: ${{env.codeCov-token}}
-          # Path to coverage file to upload
-          file: ${{env.working-directory}}/tests/unit/TestResults/coverage.opencover.xml
-          # Flag upload to group coverage metrics (e.g. unittests | integration | ui,chrome)
-          flags: unittests
-          # Environment variables to tag the upload with (e.g. PYTHON | OS,PYTHON)
-          env_vars: C#
-          # Specify whether or not CI build should fail if Codecov runs into an error during upload
-          fail_ci_if_error: true
+        run: |
```

Comment:
is there an issue number or something detailing what is broken?

Reply:
What I mean is, is this a known issue with the codecov-action?

Reply:
Sort of, I have a few threads of people with the same issue with no resolution. Will post below. Think my most recent changes screwed something up though. Added some flags and messed up the syntax.

Reply:
https://github.com/codecov/codecov-action/issues/58

Reply:
you can probably test the curl from your local machine once you generate the testing artifacts.